new-lamassu-admin/src/components/ConfirmDialog.js

```diff
@@ -0,0 +1,83 @@
+import React, { memo } from 'react'
+import { DialogActions, DialogContent, DialogContentText, IconButton, Dialog, DialogTitle as MuiDialogTitle } from '@material-ui/core'
+import { withStyles } from '@material-ui/core/styles'
+import { Button } from '../components/buttons'
+import { TextInput } from './inputs'
+import { H4, P } from './typography'
+import { ReactComponent as CloseIcon } from '../styling/icons/action/close/zodiac.svg'
+
+const styles = theme => ({
+  closeButton: {
+    position: 'absolute',
+    right: theme.spacing(1),
+    top: theme.spacing(1)
```

Comment:
We're not currently using the theme. You can configure it to use our `spacer` variable instead of the default one.